### PR TITLE
REST API docs

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/7292
closes #7292

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: CHANGES/7292.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixing REST API docs
```
Comment (ipanova) [inline CHANGES/7292.bugfix:1]: s/Fixing/Fixed

**Comment by goosemania** [comment]
I think the changelog should be of the .doc category and not a bugfix. what do you think? @fao89 @ipanova

File: CHANGES/7292.bugfix (hunk 1 above)
Comment (goosemania) [reply CHANGES/7292.bugfix:1]: +1 and also period at the end

**Comment by ipanova** [comment]
> I think the changelog should be of the .doc category and not a .bugfix. what do you think? @fao89 @ipanova

agreed, the fix is related to docs

**Pull request merged**